src/community/hz-cluster/src/main/java/org/geoserver/cluster/GeoServerSynchronizer.java

```diff
@@ -22,6 +23,9 @@ public void initialize(ClusterConfigWatcher configWatcher) {
         this.configWatcher = configWatcher;
     }
 
+    @Override
+    public void handlePreAddEvent(CatalogBeforeAddEvent event) throws CatalogException {};
```

Comment:
Same comment as above, the interface shoudl provide a default implementation that doesn't do anything.